### PR TITLE
[TASK-10] [Test] As a user, I can submit the survey after answers all the questions

### PR DESCRIPTION
**Pull request by minhnimble**
https://github.com/minhnimble/android-surveys-list/projects/1#card-46086096

## What happened

Need to write unit tests for classes used in submit survey responses feature.
 
## Insight

**Acceptance Criteria**
- [x] Write unit tests
 
## Proof Of Work
No visible changes, just under the hood tests

### CODE REVIEW

File: app/src/test/java/co/nimblehq/ui/screen/main/surveydetails/SurveyDetailsViewModelTest.kt [hunk 1]
```diff
@@ -82,4 +83,52 @@ class SurveyDetailsViewModelTest {
...
+        // Assert
+        showLoadingObserver
+            .assertNoErrors()
+            .assertValueCount(2)
+            .assertValues(true, false)
+
+        showSuccessOverlayObserver
+            .assertNoErrors()
+            .assertValueCount(1)
+            .assertValue { it }
```
Comment (manh-t) [inline app/src/test/java/co/nimblehq/ui/screen/main/surveydetails/SurveyDetailsViewModelTest.kt:132]: redundant?

Comment (minhnimble) [reply app/src/test/java/co/nimblehq/ui/screen/main/surveydetails/SurveyDetailsViewModelTest.kt:132]: oh this test is for making sure showSuccessOverlayObserver's value is `true`, not `false` 😊

Comment (manh-t) [reply app/src/test/java/co/nimblehq/ui/screen/main/surveydetails/SurveyDetailsViewModelTest.kt:132]: ah, I'm not aware of `showSuccessOverlay` type, it is Boolean type 😅  **NVM**

**Review by manh-t — APPROVED** [review]

**Comment by sleepylee** [comment]
may have to check test result that failed the CI

**Comment by minhnimble** [comment]
@sleepylee, I just checked and fixed the issue, there is a rebase interactive problem 😓 sorry about this inconvenience. 🙏

**Review by sleepylee — APPROVED** [review]

**Pull request merged**